src/services/todoService.ts

```diff
@@ -0,0 +1,74 @@
+import { json } from "express";
+const NO_ID_MESSAGE = "id isn't in the todo list!";
+const LIST_IS_EMPTY = "todo's list is empty!";
+
+export class TodoService{
+    private todos: ListItem[] = [];
+    
+    //GET
```

Comment:
remove this comments. while it help you remember which endpoint call which function you should remeber that this file isn't aware that it as call from the controller